etc/env.d/00basic

```diff
@@ -1,6 +1,6 @@
 # /etc/env.d/00basic
 # Do not edit this file
 
-PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/bin"
+PATH="/usr/local/bin:/usr/bin:/bin:/opt/bin"
```

Comment:
Gentoo has:

```bash
PATH="/usr/local/sbin:/usr/local/bin:/usr/bin:/opt/bin"
```

I don't think we should include `/bin`. We should keep `/usr/local/sbin` if we don't symlink it above.

Reply:
Yeah, `/bin` should not be here. And `/usr/local/sbin` will need to stay, obviously.